### DISCUSSION TITLE
Anyone else finding this funny?

### DISCUSSION THREAD

**Discussion by sPalmiro** [tanujnotes/Olauncher#215] [Ideas]
Sorry if this is inappropriate to say here, but i find this funny and also kinda strange.

Firstly I'd like to say that I believe that the initial development methodology of this app hit the mark perfectly 100% in feature set and UI.  However in the google play store review, i cant help but laugh at some of the user requests and complaints.. All of which can be summed by the following statement:

"your app is simple, but can we make it complicated"

However I do understand that its the intention to keep the users happy. But I really would like to see Olauncher stay preserved in its existing functionality state.  And.. If users are ever so keen on having a "minimal" launcher that has "many" features then perhaps it would be good to branch it off as a different app maybe with some paid extensions? dunno.  I am happy though that older releases can still be downloaded.

Sorry if this offends anyone, its just sooo weird. Or am i just weird? I mean what part of "minimal" don't people understand?
It's titled "Olauncher. Minimal AF" for heavens sake!

Cheers.

**Comment by tanujnotes** [comment]
I needed this. Thank you very much! 😅
The worst part is - people praise the app, ask for more features and still won't give a 5-star rating. Not entirely sure where this entitlement comes from.

**Reply by sourbrambles** [reply]
It's especially rife in the open source world. Not sure where it comes from and why people are like that. thanks for a great app 👍
